src/main/java/com/ryanharter/auto/value/moshi/AutoValueMoshiExtension.java

```diff
@@ -64,6 +66,21 @@ public String serializedName() {
         return name;
       }
     }
+
+    public Boolean nullable() {
+      return getAnnotations().contains("Nullable");
+    }
+
+    private ImmutableSet<String> getAnnotations() {
+      ImmutableSet.Builder<String> builder = ImmutableSet.builder();
```

Comment:
This might be premature, but we should think about caching this so if we have more annotations to check we won't have to iterate them each time.